src/components/new-request/search.vue

```diff
@@ -212,7 +182,43 @@
       </v-col>
     </v-row>
 
-    <div v-if="!isFederal" class="mt-3 text-center">
+    <!-- Bullets Colin Link component. Show only if a business type is selected. -->
+    <BulletsColinLink
+      v-if="entity_type_cd"
+      :businessType="entity_type_cd"
+      :colinButton="showColinButton"
+      :showDesignation="showDesignationSelect"
+      @radioButtonChange="selectedCompanyType = $event"
+    >
+      <template v-slot:name-input-slot>
+        <NameInput
```

Comment:
I know this code is the same as before, but please call it out for verification in your ticket. Thx.

Reply:
Karim, you can resolve this when you've done it.